blazor/calendar/islamic-calendar.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Islamic Calendar in Blazor Calendar Component | Syncfusion
+description: Checkout and learn here all about Islamic Calendar in the Syncfusion Blazor Calendar component and much more.
+platform: Blazor
+control: Calendar
+documentation: ug
+---
+
+# Islamic Calendar in Blazor Calendar Component
+
+In addition to the Gregorian calendar, the Calendar control supports displaying the Islamic calendar (Hijri calendar). Islamic calendar or Hijri calendar is a lunar calendar consisting of 12 months in a year of 354 or 355 days.Users can select a date from the Islamic calendar. To know more about Islamic calendar [wikipedia](https://en.wikipedia.org/wiki/Islamic_calendar).
```

Comment:
To know more about Islamic calendar [wikipedia](https://en.wikipedia.org/wiki/Islamic_calendar). - This line is not needed